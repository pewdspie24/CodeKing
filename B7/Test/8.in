12
12 247797726
4062274 4784611
1974137 2567947
2161931 896555
6127316 5558986
3845792 7144156
7703554 4529171
5659074 7506882
2301717 290945
2633021 2371635
7808791 3624323
4417908 4718096
1895313 3213456
11 113307410
4251620 8561855
3584864 7257564
1120538 9945022
4554555 9513998
9199975 9135421
5141455 4826621
3689463 8397997
3409301 762035
7500362 3088254
5785552 5811181
9817398 6737308
22 267796161
1584744 2224949
1165700 6156542
6215562 9612680
6950089 9781070
3829495 4195311
3329319 5906772
4680520 7807141
258128 8038804
5214594 4496101
421715 7027550
4562573 1942625
6441519 7848474
3539198 6898044
3111227 7595519
3539837 6219060
1544614 3845934
3239620 1429817
8393070 9420468
1693974 326473
1592345 7262817
2600329 8610236
2157100 810614
19 566879279
3171900 9058013
9009935 3972489
8010892 4026024
563673 9254866
5059992 2150101
576195 8228148
8138321 3925884
9015354 6087709
7662058 3639463
6349210 5551890
5182520 3781002
883142 9446752
243819 1879491
4201893 5199090
9231765 9343655
2964701 9579997
3643853 7736053
563506 3062831
9834372 4984569
24 742278015
9722068 3239796
2730840 3445566
4600170 5079727
7275476 7988873
4249753 2915283
7555843 733262
6182285 3865565
125252 4417310
5128917 1156574
6355906 959792
7933317 3580125
3772637 5870473
7721087 3888984
5129829 211939
5261203 398427
2574563 7362884
8635390 2936406
814356 261644
6538667 9477967
2081606 8000177
4042812 2780442
6226294 8701071
2449255 4689919
3585618 6542867
26 64342074
2487037 2717938
6265179 3303879
6972688 9925453
3614229 9903225
1314319 3657915
1334709 1263574
3578027 623029
6757511 8546193
2582332 4259810
2114740 5561635
8425505 176811
2972504 6110708
1126888 4093063
7781910 7389497
4579849 4000119
4614692 5582845
2529746 6766336
5433726 8860547
1800006 211850
5693885 6286696
6160013 7777288
5136538 5373262
2497213 532862
8176161 7748420
3581700 2646060
1655117 6552558
20 45088668
7629727 7945513
9581583 4903784
8031976 5209539
2842223 6795373
6742598 2854355
4197624 9719287
9426271 4155039
4923565 4691034
6814862 4041068
5154732 1708798
5355889 4801560
5047054 7383875
9142373 281119
9961944 9024858
3615329 9795291
4361181 8176552
7091566 8084397
5766644 1259475
5450633 4449253
7243639 4074176
25 644830813
2943570 697206
980740 4081548
407273 7849277
1594307 2868090
5965950 8702201
8193175 9436115
4888252 1501139
3809193 6427013
4971423 5012962
7765242 4408123
7742472 3970535
3756016 7250337
8282337 6534002
8478478 3151395
6577954 8777244
9078136 1866372
9306902 143983
3590662 4895368
191649 6212178
1851858 7927007
3967080 8485053
1080630 554481
9527458 8774300
6480051 7044546
9517827 9758498
12 376425693
3191378 406859
826664 4859650
3604381 8304636
565133 1523719
9559975 700339
494959 8021756
3674898 9694767
176158 839702
4040820 2503955
6669077 6318263
7969190 857902
3148052 6886526
10 881512664
5943606 2526291
5970086 1971232
3376504 2416983
1888852 9535830
4441632 6614554
4681201 5925694
7225929 7520227
8461261 5395551
7950468 5676313
9006009 8641185
18 487280004
1625387 2443650
1316537 5754888
7219097 2592939
6313959 3741756
1538398 120352
4464038 7885980
4692020 7772281
350332 3034874
514227 1322247
1136816 5190562
9624926 1842582
1897507 1034376
9230970 7145675
3456852 2540229
9183562 8533786
4956623 5977788
989885 5469571
4651334 7233069
10 20381013
6029214 7732940
4260896 9547674
2013570 9053083
2852498 6751752
3745136 1304898
2139251 7150460
6485843 6227964
6415506 1385915
1128381 540224
7962088 5184755
