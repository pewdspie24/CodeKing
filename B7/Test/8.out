59
25
101
112
109
8
5
71
43
220
63
4
